lib/textParser/huebschFormatter.js

```diff
@@ -62,6 +62,9 @@ const makeCommaSeparatedString = (array) => {
 //  in the backend PROPERLY and store then in the DB PROPERLY
 //  (mea culpa – but hey, I wrote tests)
 export const addTtsNotice = (text) => {
+  if (!text) {
+    return 'Sie hören einen automatisch vorgelesenen Beitrag.'
```

Comment:
Maybe this should be in a constant somewhere so we only have to change it in one place if we ever want to

Reply:
good point